app/controllers/api/v1/salons_controller.rb

```diff
@@ -6,7 +6,23 @@ class SalonsController < ApplicationController
       attr_accessor :salon
 
       def index
```

Comment:
Please refactor this action logic, it's seems not good variant of implementation. Think about that